examples/named_entity_recognition/Named_Entity_Recognition_with_Pytorch_and_🔭_Galileo_Simple.ipynb

```diff
@@ -86,8 +86,8 @@
         "    return os.getenv(\"MINIMIZE_FOR_CI\", \"false\") == \"true\"\n",
         "\n",
         "if _minimize_for_ci():\n",
-        "  data[\"train\"] = data[\"train\"].select(range(1000))\n",
-        "  data[\"test\"] = data[\"test\"].select(range(1000))\n",
+        "  data[\"train\"] = data[\"train\"].select(range(500))\n",
+        "  data[\"test\"] = data[\"test\"].select(range(500))\n",
```

Comment:
just confirming, this workflow run won't be considered ci right? because we don't want to limit the number of samples for the example runs

Reply:
hey good point – so all of the ci runs, which are minimized (downsampled), are located in `test.yaml` and all regular size runs are located in `examples.yaml` so what's nice about this is we can use the same notebooks for ci and populators if we like.